Makefile

```diff
@@ -326,11 +326,25 @@ sdk_config:
 openocd:
 	openocd -f interface/cmsis-dap.cfg -f target/nrf52.cfg
 load_gdb:
-	arm-none-eabi-gdb -se _build/nrf52832_xxaa_debug.out -x debug.gdb
+	arm-none-eabi-gdb -se $(OUTPUT_DIRECTORY)/nrf52832_xxaa_debug.out -x debug.gdb
 logs: SHELL:=/bin/bash   # Use the bash shell for the logs target, as sh does not have the disown command 
 logs:
 	socat pty,link=/tmp/ttyvnrf,waitslave tcp:127.0.0.1:8000 & disown
 	picocom /tmp/ttyvnrf -b 115200
 
+package_release_app:
+	@echo Packaging: s132_nrf52_6.1.1_softdevice.hex + $(OUTPUT_DIRECTORY)/nrf52832_xxaa_release.out

```

Comment:
Can you also add a link in the README of how (from where) to download the bluetooth stack?